docs/src/tool/index.md

```diff
@@ -108,7 +139,7 @@ The output name of a library is the same as if it exported a `{!rsh} Reach.App`
 
 A `{!cmd} reach compile` usage example is available in [Overview](##over-compile).
 
-# {#ref-usage-init} @{ref("cmd", "reach init")} `reach init`
+# {#ref-usage-init} @{ref("cmd", "reach init")} reach init
```

Comment:
Why remove the "`"?

Reply:
I feel like we should either remove the ticks here or improve our CSS because the current formatting isn't great:

![image](https://user-images.githubusercontent.com/3409214/159713421-d17cd7e9-e8e1-4104-9f4d-16558a2051a2.png)

Headings should look like headings IMHO. Right now they're lost in the crowd. I also think subheadings make more sense.

Reply:
I realize you were probably too busy to respond on Slack, so I've taken a stab in 5f355dad and created DCT-161 to track this.